torchvision/ops/boxes.py

```diff
@@ -133,6 +133,88 @@ def clip_boxes_to_image(boxes: Tensor, size: Tuple[int, int]) -> Tensor:
     return clipped_boxes.reshape(boxes.shape)
 
 
+def box_cxcywh_to_xyxy(boxes: Tensor) -> Tensor:
+    """
+    Converts bounding boxes from (cx, cy, w, h) format to (x1, y1, x2, y2) format.
```

Comment:
I'm OK with using the short form `cx` and `cy` in the function name, but I think we should add an explanation in the doc string. Could you add a short description what each value means?

Reply:
Can you suggest how it should be, a bit confused here.

Reply:
Basically, we need to define what `cx`, `cy`, and so on are. For example: "`(cx, cy)` represents the center of the bounding box".

Reply:
Totally agree with @pmeier here, it would be great to be a bit more explicit on what this represents

Reply:
I added them, but I'm slightly bad at documenting. Let me know if there needs changes.

Reply:
Marking as resolved due to new API.